www/rebase/src/pages/home.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { useQuery, useSubscription } from "../gqless";
+
+
+export function Home() {
```

Comment:
@p12tic all this to eventually go to this demo...

note how the api is generated automatically from schema.
and the subscribe is juste like queries. You can decide to either get automatic updates or not. just by using query or sub hooks.